aurelia-slickgrid/src/aurelia-slickgrid/aurelia-slickgrid.ts

```diff
@@ -189,6 +190,12 @@ export class AureliaSlickgridCustomElement {
       height: `${binding.gridHeight}px`,
       width: `${binding.gridWidth}px`
     };
+
+    for (const c of this.columnDefinitions) {
```

Comment:
could you add a simple comment on top of the `for` loop to briefly describe what the loop does. This makes it easier for anyone to jump in the code at any time.